internal/datastore/spanner/reader.go

```diff
@@ -179,7 +179,7 @@ func (sr spannerReader) LookupNamespacesWithNames(ctx context.Context, nsNames [
 		ctx,
 		tableNamespace,
 		spanner.KeySetFromKeys(keys...),
-		[]string{colNamespaceConfig},
+		[]string{colNamespaceConfig, colNamespaceTS},
```

Comment:
same, do we need to consider an index here?

Reply:
see above